Added explanation of surface tag list assets (#2045)
o3de/o3de.org: content/docs/user-guide/gems/reference/environment/surface-data.md

toc: true
---

> The Surface Data Gem enables surfaces such as terrain or meshes to emit signals or tags, that communicate its surface type. These signals and tags have various uses, such as creating inclusion and exclusion areas for vegetation growth or terrain.

Reviewer: minor suggestion to add additional information.

```suggestion
The Surface Data Gem enables surfaces and meshes to emit signals or tags that communicate various attributes. These signals and tags have various uses such as creating inclusion and exclusion areas, aligning and positioning objects such as vegetation, and applying physics materials.
```

Author: Sounds good!

## Adding Surface Tag Names

New **Surface Tag** names can be added by creating a new **Surface Tag Name List** asset. This can be done with the following steps:

1. Open the **Asset Editor** from the **Tools** menu.
1. Select **File** -> **New** -> **Surface Tag Name List**.

> 1. Click the **+** button to the right of the **Surface Tag Name List** row, which will add custom **Surface Tag** name elements.

Reviewer: Adding an inline icon

```suggestion
1. Click the {{< icon "add.svg" >}} **Add** button to the right of the **Surface Tag Name List** row, which adds a custom **Surface Tag** name element.

Author: Nice!

1. Fill in the **Surface Tag** name elements with tag names you would like to use.

![Adding surface tag names.](/images/user-guide/gems/reference/surface-data/surface-data-custom-tag.png)

> Once you save your **Surface Tag Name List** asset, the tag(s) you added will now be available in any **Surface Tag** data field.

Reviewer: Suggestion for style

```suggestion
Once you save the **Surface Tag Name List** asset, the tag(s) you added are available in any **Surface Tag** data field.
```

Author: Done!
